hs_discover/public/index.html

```diff
@@ -42,6 +42,7 @@
     }
 </style>
 <input type="hidden" id="qstring" value="{{ request.GET.q }}">
+<input type="hidden" id="prefillsubject" value="{{ request.GET.subject }}">
 <div class="container" id="discover-main">
```

Comment:
Allow Resource Landing Page to call with a subject querystring param

https://github.com/hydroshare/hydroshare/issues/4057